gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -1156,10 +1160,10 @@ class VeloxTestSettings extends BackendTestSettings {
     // Not useful and time consuming.
     .exclude("SPARK-33084: Add jar support Ivy URI in SQL")
     .exclude("SPARK-33084: Add jar support Ivy URI in SQL -- jar contains udf class")
-    // ReaderFactory is not registered for format orc.
+    // overflow support needed in velox
```

Comment:
Can you help to investigate why 3.3 support the overflow and 3.4 not?

Reply:
This test was not present in 3.3 and this test is part of new feature added in spark 3.4